src/views/studentregistration/studentregistration.jsx

```diff
@@ -76,7 +75,7 @@ class StudentRegistration extends React.Component {
                 ),
                 country: formData.user.country,
                 is_robot: formData.user.isRobot,
-                classroom_id: this.props.classroomId,
+                classroom_id: this.state.classroom.id,
```

Comment:
we no longer have the url param classroomId, so get it from the classroom object that API returned